docker-compose.rollkit.yml

```diff
@@ -135,6 +138,7 @@ services:
     environment:
       # TENDERMINT_RPC_URL should be the SimApp which is acting as a substitute
       # for Celestia (with IBC Eurekea enabled).
+      - NETWORK_PRIVATE_KEY=0x3f6dbfa37045c6c8ef72441a1ddaa6c43f851b26f6fb330d694fd3b4e1d45dfc
```

Comment:
[blocking] this leaks the network private key to Github. We need to remove this line, revoke the API key and move any funds used by the API key to a new key.